scripts/west_commands/runners/canopen_program.py

```diff
@@ -105,6 +107,8 @@ def do_add_parser(cls, parser):
         parser.add_argument('--sdo-timeout', type=float, default=DEFAULT_SDO_TIMEOUT,
                             help=f'''CANopen SDO response timeout in seconds
                             (default: {DEFAULT_SDO_TIMEOUT})''')
+        parser.add_argument('--block-transfer', default=False, action='store_true',
```

Comment:
`default=False` is redundant when `action='store_true'`. Just FYI, feel free to leave this as-is.